storage/backup/backup-service/src/lib.rs

```diff
@@ -4,12 +4,38 @@
 
 mod handlers;
 
-use crate::handlers::get_routes;
-use aptos_db::AptosDB;
+use crate::handlers::{get_routes, get_routes_with_backup_accessor};
+use aptos_db::{
+    backup::backup_data_accessor::BackupDataAccessor, fast_sync_aptos_db::FastSyncStorageWrapper,
+    AptosDB,
+};
 use aptos_logger::prelude::*;
 use std::{net::SocketAddr, sync::Arc};
 use tokio::runtime::Runtime;
 
+pub fn start_backup_service_with_fast_sync_wrapper(
+    address: SocketAddr,
+    db: Arc<FastSyncStorageWrapper>,
```

Comment:
Silly question: I wonder if there's a way to avoid having to duplicate this, and instead pass in an interface, such that `start_backup_service` below can take in either the `FastSyncStorageWrapper` or the `AptosDB` implementation? Otherwise, it seems like we're doing a lot of duplication, no?

Reply:
will remove `start_backup_service` which is itself a lot of work and want to only delete it once this new function is accepted.